docs/shopify_app/sessions.md

```diff
@@ -203,6 +204,66 @@ user.with_shopify_session do
 end
 ```
 
+#### Re-fetching an access token when API returns Unauthorized
```

Comment:
Do we need to add a big disclaimer (at least for now until we have a new section for the new auth strategy) that this is only applicable for the new embedded auth strategy?

🤔 And should we actually park this doc change until we're ready to release the new auth strategy? and add it when we add the "release" docs describing this new strategy? Not sure how often this gem gets released, but it might be weird showing docs with features that hasn't been released yet?